### PR TITLE
TACLS-Config Simplify installs

### PR DESCRIPTION
**Pull request by politas**
Simplifies #4594 
Not entirely sure about the TACLS filter, though. Let's see what Jenkins says

### CODE REVIEW

**Comment by politas** [comment]
Ok, I don't have a clue what's happening. I give up.

File: NetKAN/TACLS-Config-Stock.netkan [hunk 1]
```diff
@@ -13,10 +13,8 @@
     "conflicts"      : [ { "name" : "TACLS-Config" } ],
     "install" : [
         {
-            "find"          : "GameData/ThunderAerospace/TacLifeSupport",
-            "install_to"    : "GameData/ThunderAerospace",
-            "filter"        : [ "PluginData", "Textures", "MiniAVC.dll", "TacLifeSupport.dll", "TacLifeSupport.version" ],
-            "filter_regexp" : "MM_*"
+            "find"          : "ThunderAerospace/TacLifeSupport/LifeSupport.cfg",
+            "install_to"    : "GameData/ThunderAerospace/TacLifeSupport"
         }
```
Comment (techman83) [inline NetKAN/TACLS-Config-Stock.netkan]: This needs:
```
	    "find_matches_files" : true
```

File: NetKAN/TACLS.netkan [hunk 2]
```diff
@@ -16,9 +16,9 @@
     ],
     "install" : [
         {
-            "find"          : "GameData/ThunderAerospace",
-            "install_to"    : "GameData",
-            "filter_regexp" : "LifeSupport\\.cfg$"
+            "find"       : "GameData/ThunderAerospace",
+            "install_to" : "GameData",
+            "filter"     : "LifeSupport/LifeSupport.cfg"
         }
```
Comment (techman83) [inline NetKAN/TACLS.netkan]: Also looks like this will need to be
```
"filter"     : "TacLifeSupport/LifeSupport.cfg"
```

**Comment by techman83** [comment]
LGTM, merging with thanks!

**Comment by politas** [comment]
Hmm. That should not be installing `GameData/ThunderAerospace/TacLifeSupport/LifeSupport.cfg` but it is.

**Comment by techman83** [comment]
I reckon you probably need a full path on that filter. There should be only one LifeSupport.cfg in the zip I would expect, so should filter just the file without the path. Or use a filter_regexp.

**Comment by Olympic1** [comment]
Ah, I forgot to add `"find_matches_files" : true,`, that's why it didn't work for me and why I added all those filters. I was busy for a long time (and still are), so I don't now every change that is added to CKAN.

**Comment by politas** [comment]
Yeah, unfortunately there are multiple LifeSupport.cfg files in the zip.

**Pull request merged**